relayer/src/ethereum_checkpoints/tests/mod.rs

```diff
@@ -1,29 +1,73 @@
-use crate::ethereum_beacon_client::{self, slots_batch, BeaconClient};
+use crate::ethereum_beacon_client::{slots_batch, utils, BeaconClient};
 use checkpoint_light_client::WASM_BINARY;
 use checkpoint_light_client_io::{
     ethereum_common::{
         base_types::BytesFixed,
+        beacon::SyncAggregate,
         network::Network,
         utils::{self as eth_utils, BootstrapResponse, FinalityUpdateResponse, UpdateData},
         SLOTS_PER_EPOCH,
     },
-    replay_back, sync_update,
+    sync_update,
     tree_hash::TreeHash,
     Handle, HandleResult, Init, G2,
 };
 use gclient::{EventListener, EventProcessor, GearApi, Result};
 use parity_scale_codec::{Decode, Encode};
-use tokio::time::{self, Duration};
+use tokio::{
+    sync::{Mutex, MutexGuard},
+    time::{self, Duration},
+};
+
+static LOCK: Mutex<()> = Mutex::const_new(());
 
-const RPC_URL: &str = "http://127.0.0.1:5052";
+const RPC_URL: &str = "http://34.159.93.103:50000";
```

Comment:
What's this IP?

Reply:
It is our Beacon API endpoint for Holesky